Sources/ArrayExtensions.swift

```diff
@@ -201,6 +203,16 @@ extension Array where Element: Equatable {
     }
 }
 
+extension Array where Element: Hashable {
+
+    /// EZSE: Removes all occurrences of the given object(s)
+    public mutating func removeAll(_ elements: [Element]) {
+        let elementsSet = Set(elements)
```

Comment:
Oh my bad, so the filter is on the self itself ? (I am used to filters as method calls).